src/app/features/automated-status-update/automated-status-update-config-service.ts

```diff
@@ -140,6 +145,13 @@ export class AutomatedStatusUpdateConfigService {
     entity: Entity,
     entityBeforeChanges: Entity,
   ): Promise<void> {
+    // skip if already processed this specific entity revision
```

Comment:
```suggestion
    // skip if already processed this specific entity revision
    // Note views can otherwise open multiple overlapping dialogs (because the note-details component contains three components, all sharing the same form instance)
```